### ISSUE TITLE
Wrong folder name

### ISSUE THREAD

**Issue by mileschangmoda** [moda-gov-tw/PETs-Applications#21]
```
$ bash CCF_DP_job_config_gen.sh                                                                                               
Generating job configs                                                                                          
src_job_path: tree-based/jobs/bagging_base_DP                                                                   
Traceback (most recent call last):                                                                              
  File "PETs-Applications/federated_learning/financial_fraud_detection/utils/prepare_job_config.py", line 243, in <module>
    main()
  File "PETs-Applications/federated_learning/financial_fraud_detection/utils/prepare_job_config.py", line 222, in main
    meta_config = _read_json(src_job_path / JobConstants.META_FILE)
  File "PETs-Applications/federated_learning/financial_fraud_detection/utils/prepare_job_config.py", line 55, in _read_json
    raise ValueError(f"{filename} does not exist!")ValueError: tree-based/jobs/bagging_base_DP/meta.json does not exist!
Job configs generated
```

Please rename `federated_learning/financial_fraud_detection/tree-based/jobs/bagging_base_dp` to `federated_learning/financial_fraud_detection/tree-based/jobs/bagging_base_DP`

**Comment by Hector151** [comment]
It should do ``` bash CCF_data_split_gen.sh ``` first, I think it would be solved.

**Comment by mileschangmoda** [comment]
Sure I did `bash CCF_data_split_gen.sh`

Please double check that I used `CCF_DP_job_config_gen.sh` not  `CCF_job_config_gen.sh`

https://github.com/moda-gov-tw/PETs-Applications/blob/b67bafd9f6ed6315dacb6768236a4e4be9dea447/federated_learning/financial_fraud_detection/utils/prepare_job_config.py#L90-L94

In the code L93, if I pass argument --DP, the job define should be write in `tree-based/jobs/bagging_base_DP/meta.json`.

**Comment by Hector151** [comment]
I have modified the L94 to strDP = "_dp". Thanks for the advise.

**Comment by haeter525** [comment]
This issue has been fixed by 5becf99. Thank @Hector151.
